commit 616cfbf30b6e41f03f8c647907f23dca3a999abf
Author: Kees Cook <kees@kernel.org>
Date:   Thu Feb 22 14:00:48 2024 -0800

    MAINTAINERS: Update LEAKING_ADDRESSES details
    
    Tobin hasn't been involved lately, and I can step up to be a reviewer
    with Tycho. I'll carry changes via the hardening tree.
    
    Reviewed-by: Tycho Andersen <tandersen@netflix.com>
    Link: https://lore.kernel.org/r/20240222220053.1475824-1-keescook@chromium.org
    Signed-off-by: Kees Cook <keescook@chromium.org>

diff --git a/MAINTAINERS b/MAINTAINERS
index 216d02a3fed5..cd651c4df019 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -12154,11 +12154,11 @@ F:	Documentation/scsi/53c700.rst
 F:	drivers/scsi/53c700*
 
 LEAKING_ADDRESSES
-M:	Tobin C. Harding <me@tobin.cc>
 M:	Tycho Andersen <tycho@tycho.pizza>
+R:	Kees Cook <keescook@chromium.org>
 L:	linux-hardening@vger.kernel.org
 S:	Maintained
-T:	git git://git.kernel.org/pub/scm/linux/kernel/git/tobin/leaks.git
+T:	git git://git.kernel.org/pub/scm/linux/kernel/git/kees/linux.git for-next/hardening
 F:	scripts/leaking_addresses.pl
 
 LED SUBSYSTEM